RuleDocumentation/AvoidAssignmentToAutomaticVariable.md

```diff
@@ -0,0 +1,33 @@
+# AvoidAssignmentToAutomaticVariable
+
+**Severity Level: Warning**
+
+## Description
+
+`PowerShell` exposes some of its build in variables that are known as automatic variables. Many of them are read-only and PowerShell would throw an error when trying to assign an value on those. The remaining automatic variables should only be assigned to in certain special cases to achieve a certain effect as a special technique.
```

Comment:
*built-in* rather than "build in"
"the remaining" would probably be better as *other*